src/components/Pagination.tsx

```diff
@@ -0,0 +1,79 @@
+import { Dispatch, SetStateAction, useMemo } from "react";
+import { IconButton } from "src/components";
+import { Css, Palette } from "src/Css";
+import { SelectField } from "src/inputs";
+import { useTestIds } from "src/utils";
+
+export type PageSettings = {
+  page: number;
+  perPage: number;
+};
+
+type PaginationProps = {
+  label: string;
+  totalCount: number;
+  settings: PageSettings;
+  setSettings: Dispatch<SetStateAction<PageSettings>>;
+};
+
+// Make a list of 50/100/150/etc page sizes for the user to chose
+export const pageOptions = Array(5)
+  .fill(0)
+  .map((_, i) => ({ id: (i + 1) * 50, name: ((i + 1) * 50).toString() }));
+
+export function Pagination(props: PaginationProps) {
+  const { label, settings, totalCount, setSettings } = props;
+  const { perPage, page } = settings;
+
+  const hasNextPage = useMemo(() => page < totalCount / perPage, [page, perPage, totalCount]);
+
+  const tid = useTestIds(props, "pagination");
+  return (
+    <div css={Css.df.bGray300.bt.xs.gray500.px1.ml2.pt2.$} {...tid}>
+      <div css={Css.df.mya.mr2.$} {...tid.perPageLabel}>
+        {label} per page:
+      </div>
+      <div css={Css.wPx(78).$}>
+        <SelectField
```

Comment:
**suggestion:** I think you might want to use the `compact` variant of the `SelectField`. When looking at the mocks, they use a SelectField that has a 32px height, which would signal it is the `compact` style.
https://www.figma.com/file/aWUE4pPeUTgrYZ4vaTYZQU/%E2%9C%A8Beam-Component-Library?node-id=32601%3A99960&t=omqHcE1hZIcy6vQz-0

Reply:
You're right, fixed 👍